bzt/resources/base-config.yml

```diff
@@ -156,6 +156,11 @@ services:
     - conn-all
 
 modules: # TODO: cleanup as much as possible from defaults
+  blazemeter:
+    dedicated-ips: false
```

Comment:
But having these fields in base config does not guarantee it. User can upload config through GUI and there will be no fields.  Or user will use direct API calls to arrange the test.
Server side has to handle defaults for these fields on its own, because of GUI flow and direct API flow.